css/properties/visibility.json

```diff
@@ -4,6 +4,7 @@
       "visibility": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/visibility",
+          "spec_url": "https://drafts.csswg.org/css-flexbox-1/#visibility-collapse",
```

Comment:
This is also adding requirements and not defining `visibility:collapse`, but still it seems more reasonable to include that the previous examples in this PR I was skeptical of. It should be linked from the "collapse" subfeature of this entry, though, not here.

Reply:
```suggestion
          "spec_url": "https://drafts.csswg.org/css2/#propdef-visibility",
```

Reply:
Not https://drafts.csswg.org/css2/#visibility?

Reply:
```suggestion
          "spec_url": "https://drafts.csswg.org/css2/#visibility",
```